Project/Assets/ML-Agents/Examples/Worm/Scripts/WormAgent.cs

```diff
@@ -1,107 +1,137 @@
 using UnityEngine;
 using Unity.MLAgents;
+using Unity.Barracuda;
 using Unity.MLAgents.Actuators;
 using Unity.MLAgentsExamples;
 using Unity.MLAgents.Sensors;
 
 [RequireComponent(typeof(JointDriveController))] // Required to set joint forces
 public class WormAgent : Agent
 {
-    [Header("Target To Walk Towards")]
-    [Space(10)]
-    public Transform target;
-
-    public Transform ground;
-    public bool detectTargets;
-    public bool targetIsStatic;
-    public bool respawnTargetWhenTouched;
-    public float targetSpawnRadius;
-
-    [Header("Body Parts")]
-    [Space(10)]
-    public Transform bodySegment0;
+    //The type of agent behavior we want to use.
+    //This setting will determine how the agent is set up during initialization.
+    public enum WormAgentBehaviorType
+    {
+        WormDynamic,
+        WormStatic
+    }
+
+    [Tooltip(
+        "Dynamic - The agent will run towards a target that changes position.\n\n" +
+        "Static - The agent will run towards a static target. "
+    )]
+    public WormAgentBehaviorType typeOfWorm;
+
+    const float m_maxWalkingSpeed = 10; //The max walking speed
+
+    //Brains
+    //A different brain will be used depending on the CrawlerAgentBehaviorType selected
+    [Header("NN Models")] public NNModel wormDyBrain;
```

Comment:
Please do not use Brain here. We are trying to remove all mentions of Brain. I would use `wormDynModel` since it is a model.

Reply:
changed to use `wormDynModel` per your suggestion